### ISSUE TITLE
Vanished feature? / Feature Request: Progress in Percent

### ISSUE THREAD

**Issue by annomatik** [picotorrent/picotorrent#986]
If I recall correctly, in .18, the Progress was not only shown as a green bar, but also as a number like 80.7 % on top of the bar. Same for the progress of files within the torrent.

Would it be possible to bring back that feature (does not seem to be available in .22), or, if I was just imagining it, implement that? That would make it a lot easier for me to interpret the progress bar, especially if the bar is "almost" at the end.

Thanks!

**Comment by vktr** [comment]
Yeah, we used to have that in the Qt-based versions. wxWidgets does not provide that kind of progress bar, but I'm currently reimagining how the progress bar should look like, so this fits well within that feature.
